common/src/traits.rs

```diff
@@ -329,15 +329,15 @@ impl<DEXId, AccountId, AssetId> LiquiditySource<DEXId, AccountId, AssetId, Fixed
     }
 
     fn quote_weight() -> Weight {
-        Weight::zero()
+        Weight::from_all(1)
     }
 
     fn exchange_weight() -> Weight {
-        Weight::zero()
+        Weight::from_all(2)
     }
 
     fn check_rewards_weight() -> Weight {
-        Weight::zero()
+        Weight::from_all(3)
```

Comment:
Why do you need non-zero values?

Reply:
So that tests actually test something. I.e. if weight calculation does not add weight of some liquidity source, it is not spotted with zero weight because adding zero is no-op.

Reply:
All tests are passed with zero values.
Don't change the trait only for tests